utils/generic-components.stanza

```diff
@@ -120,30 +175,24 @@ public pcb-component gen-cap-cmp (cap:Double, tol:Double, max-v:Double, pkg:Stri
     capacitor => true
     type => "ceramic"
     capacitance => cap
-    tolerance => (tol * 0.01)
```

Comment:
I looked up db-parts:
- tolerance is in % in the emodels as a scalar (symmetric interval)
```
defn emodel-pourcentage-tolerance (value: MinMaxRange|False) -> Double|UNKNOWN :  ; FIXME: Tolerance minmax range not handled properly but emodel is going depricated
  match(value: MinMaxRange) :
    100. * max(value)
  else:
    UNKNOWN
```
- tolerance in properties is only defined for Inductor right now as a tuple unitless:
```
match(tolerance: MinMaxRange): tolerance => [min(tolerance), max(tolerance)]
```
So there is a decision to make on keeping the tuple or a scalar to keep a symmetric interval.

It is a tuple because propoerties only accept primitive types (no MinMaxRange) but there will be a new esir anyway.

Reply:
gen-cap-cmp is deprecated for the time being, so I'm not worried about compatibility.